notebooks/ai-ml/HuggingFaceEmbeddingGenerationInSpark.ipynb

```diff
@@ -0,0 +1,772 @@
+{
```

Comment:
<p>Enable <a href="&nbsp;https://cloud.google.com/dataproc-serverless/docs/concepts/autoscaling" target="_blank">autoscaling</a> by setting the following parameters in Spark properties:</p>

<pre class="ql-syntax" spellcheck="false">spark.dynamicAllocation.enabled = true
spark.dynamicAllocation.maxExecutors = 100
spark.dynamicAllocation.minExecutors = 5
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudDataproc/cloud-dataproc/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>